### PR TITLE
Feature/map messages user

### PR DESCRIPTION
**Pull request by jcgsxr**

### CODE REVIEW

File: Sqwiggle.podspec [hunk 1]
```diff
@@ -14,5 +14,6 @@ Pod::Spec.new do |s|
 
   s.dependency 'AFNetworking', "~> 2.0.3"
   s.dependency "ObjectiveSugar", '~> 0.9'
+  s.prefix_header_contents = '#import <ObjectiveSugar/ObjectiveSugar.h>', '#import <AFNetworking/AFNetworking.h>'
```
Comment (cameronwebbable) [inline Sqwiggle.podspec:17]: Out of pure curiosity (since I've never used prefix_header_contents before), why is this here? And why do we have to list same things in SqwiggleSDK-Prefix.pch ?

File: iOSSDK/SQObject.h [hunk 2]
```diff
@@ -17,7 +17,7 @@
 @property (nonatomic, strong) NSNumber *ID;
 
 //Relative URL for object in API. NOTE: This must be overriden in each object.
-@property (nonatomic, readonly) NSString *relativeURL;
+@property (nonatomic, copy) NSString *relativeURL;
```
Comment (cameronwebbable) [inline iOSSDK/SQObject.h:20]: Good call. Leads me to also think that we should probably drop readonly from all properties in SDK (except APIKey).

File: Sqwiggle.podspec (hunk 1 above)
Comment (jcgsxr) [inline Sqwiggle.podspec:17]: It's standard practice (albeit slightly outdated) to place imports for commonly used libs in the prefix file. This effectively grants the entire project the imports automatically when they're placed there, which is why it's placed in the sdk's prefix file `SqwiggleSDK-Prefix.pch`.

As for the the podspec, it will place those imports in the generated prefix file for the pod since the sdk's prefix file doesn't get copied over when running `pod install`

Comment (cameronwebbable) [inline Sqwiggle.podspec:17]: Ah, that makes sense. Appreciate clarification.
